packages/console/src/components/Checkbox/index.tsx

```diff
@@ -8,16 +8,16 @@ type Props = Omit<InputHTMLAttributes<HTMLInputElement>, 'type'> & {
   label?: ReactNode;
 };
 
-const Checkbox = ({ label, disabled, ...rest }: Props) => {
+const Checkbox = ({ label, disabled, ...rest }: Props, ref: LegacyRef<HTMLInputElement>) => {
```

Comment:
The type can just be `Ref<HTMLInputElement>`

Reply:
updated